code/sonalyze/db/persistentcluster.go

```diff
@@ -12,7 +12,8 @@
 //
 // In each (leaf) directory there are lists of files matching specific semantic patterns: "*.csv"
 // filtered for some weird cases for `sonar ps` samples, "sysinfo-*.json" for `sonar sysinfo` data,
-// and so on.  These lists are populated lazily when the files in the directory are first needed.
+// and so on.  (These are older data files.  Newer data files follow new naming patterns.)  These
+// lists are populated lazily when the files in the directory are first needed.
```

Comment:
This comment should reference the doc block FILE NAME SCHEMES in db/clusterstore.go.